content/en/blog/_posts/2025-05-12-ensure-secret-pulled-images/index.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: blog
+title: "Kubernetes v1.33: Image Pull Policy the way you always thought it worked!"
+date:  2025-05-12
```

Comment:
Dated for the [publication date suggested in slack via the publication schedule](https://kubernetes.slack.com/archives/CJDHVD54J/p1745461641777509).